pkg/parser/additional/certificate_utils.go

```diff
@@ -0,0 +1,79 @@
+package additional
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+// CheckCertificateBody verifies if the attribute 'certificate_body' refers a file
+func CheckCertificateBody(content string) (b bool, s string) {
+	var re = regexp.MustCompile(`[0-9a-zA-Z-/\\_]+\.pem`)
+	match := re.FindAllString(content, -1)
+
+	if match == nil {
+		return false, ""
+	}
+
+	return true, match[0]
+}
+
+func getCertificateInfo(filePath string) (a [3]int, i int) {
+	certPEM, err := os.ReadFile(filePath)
+
+	if err != nil {
+		panic("Failed to read the file " + filePath)
```

Comment:
We shouldn't be panicking here, maybe create a struct that takes in a and i return values and return the struct and the error 

```
type returnStruct struct {
   a [3]int
   i int
}

func getCertificateInfo(filePath string) (returnStruct, error) {
	certPEM, err := os.ReadFile(filePath)

	if err != nil {
		return returnStruct{}, err
```

Reply:
Thank you so much for your suggestions, João 😄